db/migrate/20150906173134_create_teams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.belongs_to :account
+      t.belongs_to :team
```

Comment:
Yes. It needs to be named `team` in the actual migration because `parent_team` is not a model or table in the database. In the model, we can create an alias for `parent` to access it, though. You're right about the second part, I neglected to add those.